### ISSUE TITLE
Invites don't local echo.

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-x-ios#1189]
### Steps to reproduce

1. Go to a room
2. Invite a user to it
3. See absolutely no evidence in the member list that you've invited a user (especially if messages are delayed due to #1105 or bad network connectivity)
4. Feel completely disoriented

#### What did you expect?

You should see a local echo of the invite in the membership list, even if you haven't seen the remote echo of the invite yet.  It could have a spinner or be greyed out or something to show that it's still sending.

#### What happened instead?

A feeling of extreme flakiness and nausea.

### Application version

254

### Will you send logs?

No

**Comment by kittykat** [comment]
@ara4n can you reproduce it after clearing cache?

I had a similar issue two weeks ago which was caused by https://github.com/matrix-org/matrix-rust-sdk/issues/2078

**Comment by ara4n** [comment]
I cleared cache, and can still repro it when inviting a new user.

**Comment by jplatte** [comment]
I don't think this is actually possible. EleWeb doesn't seem to create a local echo either. The [invite](https://spec.matrix.org/latest/client-server-api/#post_matrixclientv3roomsroomidinvite) endpoint does not have a transaction ID parameter that could be used to identify the event as belonging to the invite request (also, the invite response does not contain the event ID either).

**Comment by manuroe** [comment]
I am deprioritising this feature because 1/ it would require specific work if we do not change the backend API and 2/ it does not exist elsewhere yet.

**Comment by manuroe** [comment]
Tested it today. The member list refreshes at some point after, I guess, the backend acknowledges the request. The refresh can be instant or multi seconds according to network or matrix.org conditions.

Managing a local echo with all the retry mechanisms behind to make it reliable is a significant amount of work.

Why not displaying a banner like the one when the user is removing another member?

![Image](https://github.com/user-attachments/assets/937810e9-7fa3-4655-b064-2a0368fb168d)

**Comment by ara4n** [comment]
also, the problem isn't just lack of local echo - but remote echo too
